tsconfig.json

```diff
@@ -2,13 +2,23 @@
   "compilerOptions": {
     "declaration": true,
     "importHelpers": true,
-    "module": "commonjs",
+    "module": "ES2022",
+		"moduleResolution": "node16",
     "outDir": "lib",
     "rootDir": "src",
-    "strict": true,
-    "target": "es2017",
-    "esModuleInterop": true
+    "target": "esnext",
+    "lib": [
+			"ES2021"
+		],
+    "esModuleInterop": true,
+    "noPropertyAccessFromIndexSignature": false,
+    "isolatedModules": true
   },
   "include": ["src/**/*"],
-  "exclude": ["node_modules", "test", "src/**/*.test.ts"]
+  "exclude": ["node_modules", "test", "src/**/*.test.ts"],
+  // "ts-node": {
+  //   "transpileOnly": true,
+  //   "files": true,
+  //   "experimentalResolver": true
+  // }
```

Comment:
Worth removing instead of commenting out?

Reply:
Definitely.